modules/social_features/social_event/modules/social_event_managers/src/Plugin/views/field/SocialEventManagersViewsBulkOperationsBulkForm.php

```diff
@@ -347,17 +347,15 @@ public function getEntityLabel(EntityInterface $entity) {
     /** @var \Drupal\profile\Entity\ProfileInterface $profile */
     $profile = reset($profiles);
 
-    // It must be a Guest so we pick the name values we can get.
+    // It must be a Guest, so we pick the name values we can get.
     if (!$profile) {
       $name = '';
-      $first_name = $entity->get('field_first_name')->getValue()[0]['value'];
-      $last_name = $entity->get('field_last_name')->getValue()[0]['value'];
-      if (!empty($first_name)) {
-        $name .= $first_name;
+      if (!empty($first_name = $entity->get('field_first_name')->getValue())) {
+        $name .= $first_name[0]['value'];
         $name .= ' ';
```

Comment:
If `field_last_name` is empty we will have a name with space on the right side. I'd recommend `trim` the name before return.